atom/browser/api/atom_api_top_level_window.cc

```diff
@@ -71,6 +71,21 @@ v8::Local<v8::Value> ToBuffer(v8::Isolate* isolate, void* val, int size) {
 
 }  // namespace
 
+namespace {
```

Comment:
There is already an anonymous namespace right above, not need to create a new one.

Reply:
Feels bad... Didn't even realize 😆